src/app/header/header.component.html

```diff
@@ -1,14 +1,24 @@
-<md-toolbar [ngStyle]="{padding:0}">
-  <div class="mat-toolbar">
-    <md-toolbar-row class="header">
-      <button md-icon-button [ngStyle]="{backgroundColor: '#cacaca'}"><md-icon>menu</md-icon></button>
+<md-toolbar class="header">
+  <div class="mat-toolbar header">
+    <md-toolbar-row class="mat-toolbar-row header">
+      <button md-icon-button [mdMenuTriggerFor]="appMenu"><md-icon>menu</md-icon></button>
```

Comment:
Add padding for the toolbar
![image](https://user-images.githubusercontent.com/14135364/29769086-ffc293a4-8c12-11e7-9698-f64e87ea5271.png)